lib/typing/snowflake.go

```diff
@@ -6,11 +6,9 @@ import (
 	"github.com/artie-labs/transfer/lib/typing/ext"
 )
 
-// https://docs.snowflake.com/en/sql-reference/intro-summary-data-types.html
-
-func SnowflakeTypeToKind(snowflakeType string) KindDetails {
-	snowflakeType = strings.ToLower(snowflakeType)
-
+// snowflakeTypeToKind - Converts a Snowflake type to a KindDetails.
+// Following this spec: https://docs.snowflake.com/en/sql-reference/intro-summary-data-types.html
+func snowflakeTypeToKind(snowflakeType string) KindDetails {
```

Comment:
We can probably also get rid of the `strings.ToLower` here: https://github.com/artie-labs/transfer/pull/432/files#diff-e6f658cbda168f6e7a233f8bcf5807e9b37544c785787f3a1c4e3890c506b681R25

Reply:
Good catch!